### ISSUE TITLE
Milo object size

### ISSUE THREAD

**Issue by emdann** [MikeDMorgan/miloR#4]
Any idea why the `Milo` object is much bigger than the `SingleCellExperiment` object, even before computing all the graphs and neighborhoods?
```r
> sce
class: SingleCellExperiment 
dim: 33694 50514 
metadata(0):
assays(2): counts logcounts
rownames(33694): TSPAN6 TNMD ... RP11-107E5.4 RP11-299P2.2
rowData names(0):
colnames(50514): FCAImmP7179369-AAACCTGAGCCCAATT FCAImmP7179369-AAACCTGAGCCTATGT ... Human_colon_16S7985397-TTTGCGCAGAGCCCAA
  Human_colon_16S7985397-TTTGGTTAGTACGCGA
colData names(14): Sample donor ... cell types umap_density_Age
reducedDimNames(1): MOFA
altExpNames(0):
> object.size(sce)
16486280 bytes
> milo <- Milo(sce)
> milo
class: Milo 
dim: 33694 50514 
metadata(0):
assays(1): expression
rownames(33694): TSPAN6 TNMD ... RP11-107E5.4 RP11-299P2.2
rowData names(0):
colnames(50514): FCAImmP7179369-AAACCTGAGCCCAATT FCAImmP7179369-AAACCTGAGCCTATGT ... Human_colon_16S7985397-TTTGCGCAGAGCCCAA
  Human_colon_16S7985397-TTTGGTTAGTACGCGA
colData names(14): Sample donor ... cell types umap_density_Age
reducedDimNames(1): MOFA
altExpNames(0):
neighbourhoods dimensions(1): 0
neighbourhoodCounts dimensions(2): 1 1
neighbourDistances dimensions(2): 1 1
graph names(0):
> object.size(milo)
13639413808 bytes
```

Not sure if this is a real issue, but it seems to make things like subsetting very slow and leads to frequent R crashes.

**Comment by MikeDMorgan** [comment]
Hmm, I can only assume that the SCE is using something under the hood that isn't inherited in the Milo object. This may take some time...

**Comment by MikeDMorgan** [comment]
The Milo object now faithfully recapitulates the SCE object, fixed in commit cb2882cf25c3e78a4a08038c991ccf0fc69553c2.
```
> object.size(my.sce)
7680736 bytes
> mylo <- Milo(my.sce)
> object.size(mylo)
7683344 bytes
```
